properties.xml

```diff
@@ -20,8 +20,9 @@
   </game>
   
   <server-settings>
-	<port>10000</port>
-	<portws>9998</portws>
+    <port>10000</port>
+    <portws>9998</portws>
+    <porthttp>10001</porthttp>
```

Comment:
probably do want to notate that 10001 is used in beeta
<port>10002</port>
<portws>10001</portws>

Reply:
Am I getting this right:
|Env|Telnet|Websocket|HTTP|
|-|-|-|-|
|Production | 10000 | 9998 | choose |
|Beta | 10002 | 10001 | choose |
|Local dev | 10000 | 9999 | choose |

Please correct me and suggest ports for HTTP

Reply:
Close prod is using 9999 for the websocket, I probably left 9998 in there from testing locally the properties file gets generated during deployment but just want to make a note in the readme about beta really

 since 80 and 442 and currently being used by the UI probably need to use a semi non standard port something like 8080 or 4443 or can use something starting around 9000 I'd don't have a strong opinion on it.

Reply:
I'll just take the next ports in the series then

|Env|Telnet|Websocket|HTTP|
|-|-|-|-|
|Production | 10000 | 9999 | 9998|
|Beta | 10002 | 10001 | 10003 |
|Local dev | 10000 | 9999 | 9998 |